tests/integrationv2/common.py

```diff
@@ -142,3 +244,15 @@ def __init__(self,
         self.use_client_auth = use_client_auth
         self.client_certificate_file = client_certificate_file
         self.client_key_file = client_key_file
+
+# Common curves
+TLS_CURVES = [
+    Curves.P256,
+    Curves.P384
+]
+
+
+# TLS1.3 specific curves
+TLS13_CURVES = [
+    Curves.X25519,
+]
```

Comment:
Note that Curve X25519 is not supported in libcrypto versions 1.0.2 or lower.

Reply:
Good call out. I still need to update these structure to work with different versions of OpenSSL.